src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
@govarguz suggesting you implement `connect/disconnect` and `removeAll` separately as suggested by @MrTheodor, will they have some identical functions inside (like `sigBeforeSend`, etc.)?